packages/debug-utils/index.js

```diff
@@ -39,8 +39,8 @@ const commandReference = {
   "u": "step out",
   "n": "step next",
   ";": "step instruction (include number to step multiple)",
-  "p": "print instruction & state (`p [mem|cal|sto]*`; see docs for more)",
-  "l": "print additional source context",
+  "p": "print instruction & state (`p [mem|cal|sto|line-number]*`; see docs for more)",
```

Comment:
Again, I don't think this conveys what we want.  Possibly we can just leave this out of the quick help -- I feel like this is a somewhat niche use case.  Is the quick help even long enough to accommodate this?  Like I think this sould be something more like `p [mem|cal|sto]* [+<instructions-ahead>] [-<instructions-back>]`, but that's getting fairly long... we could abbreviate the word "instructions" to "instrs" or something, not sure offhand how much that helps...

Reply:
The help looks better with `instrs`:
```
(g) turn on generated sources, (G) turn off generated sources except via `;`
(p) print instruction & state (`p [mem|cal|sto|]* [+<instrs-ahead>] [-<instrs-back>]`; see docs for more)
(l) print additional source context (`l [+<lines-ahead>] [-<lines-back>]`; see docs for more)
```

Reply:
Nice!

Reply:
OK trying it out these are still too long I'm afraid... want these to fit in an 80 character width I think.

For `l` this is easy to fix -- we don't need the "see docs for more" note because that's the whole syntax!  That note only exists on `p` because we couldn't fit the whole syntax on one line.

For `p`... I think we're just going to have to drop the bit about +/-, and just leave that in the online documentation; I don't think there's any way to make that fit.

Reply:
Come to think of it, the "see docs for more" note on `v` doesn't need to be there either, does it?  Dunno if getting rid of that needs to go in this PR, but it should be removed at some point.  (I can pick that up later if you don't have time to get it.)

Reply:
Thanks!